docker-compose.yml

```diff
@@ -114,6 +115,7 @@ services:
       PUBLIC_HASURA_CLIENT_URL: http://localhost:8080/v1/graphql
       PUBLIC_HASURA_SERVER_URL: http://hasura:8080/v1/graphql
       PUBLIC_HASURA_WEB_SOCKET_URL: ws://localhost:8080/v1/graphql
+      PUBLIC_SEQUENCING_MODE: "templating"
```

Comment:
We should figure out which templating mode should be the default in what environments (e.g. development, new users, existing users)

Reply:
`"legacy"` has now been updated as the default here and in the ui!